client/my-sites/plans/jetpack-plans/storage-pricing-header/style.scss

```
@@ -0,0 +1,26 @@
+.storage-pricing-header {
+	text-align: center;
+	margin-bottom: 50px;
+}
+
+.storage-pricing-header__title {
+	.formatted-header__title {
+		font-weight: 700;
+		font-size: $font-headline-medium;
```

Let's reduce letter spacing here to match the design mocks.

```suggestion
	.formatted-header__title {
		font-weight: 700;
		font-size: $font-headline-medium;
		letter-spacing: -0.95px;
```